File: spring/services/server/src/main/java/io/mendirl/spring/server/ServerApplication.java
package io.mendirl.spring.server;

import org.springframework.boot.ApplicationRunner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
import org.springframework.context.annotation.Bean;

@SpringBootApplication
@ConfigurationPropertiesScan
public class ServerApplication {

    public static void main(String[] args) {
        SpringApplication.run(ServerApplication.class, args);
    }

    @Bean
    ApplicationRunner runner(RetrieverService service) {
        return args -> service.showProperties();
    }
}

File: spring/services/server/src/main/kotlin/io/mendirl/spring/services/server/ServerApplication.kt
package io.mendirl.spring.services.server

import de.codecentric.boot.admin.server.config.EnableAdminServer
import io.mendirl.spring.services.common.JupiterProperties
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication

@SpringBootApplication
@EnableAdminServer
@EnableConfigurationProperties(JupiterProperties::class)
class ServerApplication

fun main(args: Array<String>) {
    runApplication<ServerApplication>(*args)
}
